src/components/molecules/Card.js

```diff
@@ -5,12 +5,15 @@ import "../../css/molecules/card.css";
 function Card({title, description, children}) {
     return (
         <article className="card">
-            <h2 className="card-title">
-                {title}
-            </h2>
-            <p className="card-description">
-                {description}
-            </p>
+            {title &&
+                <h2 className="card-title">
+                    {title}
+                </h2>}
+            {description &&
+                <p className="card-description">
+                    {description}
+                </p>
+            }
```

Comment:
Maybe instead of being empty, it should say "Untitled article" and "This article contains no content."  This would probably be a rare occurrence anyways since such articles should get rejected from being created.

Reply:
`<Card />` is not unique to articles, the reason it is optional is that you may not always need it (i.e. with `<ProfileHeader />`).